tests/test_acado.py

```diff
@@ -38,10 +38,24 @@ def test_expression_add(self):
 
         self.assertIsInstance(y, Expression)
 
-        y = 0.5 + x 
+        y = 0.5 + x
 
         self.assertIsInstance(y, Expression)
 
+    # def test_expression_division(self):
```

Comment:
Is there a reason for committing commented out code?  Should this be an expected failure or skipped test (use the appropriate decorator, if it is).

Reply:
This is an expected failure, but one I'm trying to fix today!

Reply:
This test now passes, so uncommented.